README.md

```diff
@@ -48,7 +48,7 @@ In publications, please use the following publication to cite Octo-Tiger:
 
 *  Dominic C. Marcello, Sagiv Shiber, Orsola De Marco, Juhan Frank, Geoffrey C. Clayton, Patrick M. Motl, Patrick Diehl, Hartmut Kaiser, "Octo-Tiger: A New, 3D Hydrodynamic Code for Stellar Mergers that uses HPX Parallelisation", accepted for publication in the Monthly Notices of the Royal Astronomical Society, 2021
 
-For more publications, refer to Octo-Tigers' [documentation](https://stellar-group.github.io/octotiger/doc/html/md_content_publications.html).
+For more publications, please review Octo-Tigers' [documentation](http://octotiger.stellar-group.org/papers/).
```

Comment:
Actually, why http instead of https? Not that we technically need https for a website like that, but why suffer the "not secure" warnings if the https version is working as well?

Reply:
Good catch. It is fixed.